src/foam/dao/EasyDAO.js

```diff
@@ -368,9 +368,13 @@ foam.CLASS({
           delegate = new foam.dao.PMDAO.Builder(getX()).setCSpec(getCSpec()).setDelegate(delegate).build();
 
         for ( Indexer i : indexes ) {
+          DAO d = getMdao();
```

Comment:
Why is this necessary? cmd_ should flow down the decorator chain util the mdao picks it up.